src/scripts/globalize.js

```diff
@@ -243,7 +243,7 @@ import * as userSettings from './settings/userSettings';
 
     updateCurrentCulture();
 
-    Events.on(userSettings, 'change', function (e, name) {
+    Events.on(userSettings.currentSettings, 'change', function (e, name) {
```

Comment:
Oops, good catch.
You can import it as `import { currentSettings as userSettings } from ...` instead of this.
That bunch of exports is for backward compatibility when we were migrating to ES6, iirc.